web-console/knikubevirt/Create-vm/step-2-networking/create-vm-networking.md

```diff
@@ -3,17 +3,20 @@
 ## Step 2: Networking
 
 ![Default](img/3-00.png)
-The user enters the Networking step with an existing NIC pre-defined, set to Type (local). In this step, the 'Review and Create' button is still available to skip the rest of the steps. At this point, the user can select to customize this NIC, add new ones, or remove any they wish.
+The user enters the Networking step with an existing  Network interface pre-defined, set to Type (local). In this step, the 'Review and Create' button is still available to skip the rest of the steps. At this point, the user can select to customize this Network interface, add new ones, or remove any they wish.
 
 ![Adding NIC](img/3-01.png)
-Clicking on 'Add NIC' will pop up a "Create NIC" modal. The NIC name is prepopulated with a numeric count form of existing NICs on this flow. The user may change the NIC name. All the fields are preselected except Type. In order for the user to apply and add this NIC, the user must select one of the type options.
+Clicking on 'Add  Network interface' will pop up a "Add Network interface" modal. The Network interface name is prepopulated with a numeric count form of existing Network interfaces on this flow but the user may change it. In order for the user to apply and add this Network interface, they must select one of the type options.
+In case the user selects 'Masquerade' on the Type field, they can add multiple Ports.
+In that case the user will get a list of Ports (port name, port number, protocol) which will be then forwarded and exposed to the virtual machine. The ports will be displayed in rows so the user can add or remove them.
 
-![Type - Masquerade](img/3-02.png)
-In case the user selects Masquerade on the Type field, new indented fields will appear.
+![masquerade is selected](img/3-03.png)
```

Comment:
Similar to the "Add CD" option we are adding...should the +Add port be underneath the first Port listed?

Reply:
@lizsurette yes, I also think it makes sense and it was the original place for it, but then there was a request to follow the consol's pattern so it was moved.

Reply:
Yes @lizsurette that was the exact original place for it, but then I misunderstood a comment and changed it. Now placing back. Sorry about that.